diff --git a/node_modules/tshy/dist/esm/build-commonjs.js b/node_modules/tshy/dist/esm/build-commonjs.js
index a84ddb3..ef414e0 100644
--- a/node_modules/tshy/dist/esm/build-commonjs.js
+++ b/node_modules/tshy/dist/esm/build-commonjs.js
@@ -1,6 +1,6 @@
 import chalk from 'chalk';
 import { spawnSync } from 'node:child_process';
-import { renameSync, unlinkSync } from 'node:fs';
+import { renameSync, unlinkSync, existsSync } from 'node:fs';
 import { relative, resolve } from 'node:path/posix';
 import buildFail from './build-fail.js';
 import config from './config.js';
@@ -26,8 +26,12 @@ export const buildCommonJS = () => {
         for (const [override, orig] of pf?.map.entries() ?? []) {
             const stemFrom = resolve(`.tshy-build/${d}`, relative(resolve('src'), resolve(override))).replace(/\.cts$/, '');
             const stemTo = resolve(`.tshy-build/${d}`, relative(resolve('src'), resolve(orig))).replace(/\.tsx?$/, '');
-            unlinkSync(`${stemTo}.js.map`);
-            unlinkSync(`${stemTo}.d.ts.map`);
+            if (existsSync(`${stemTo}.js.map`)) {
+                unlinkSync(`${stemTo}.js.map`);
+            }
+            if (existsSync(`${stemTo}.d.ts.map`)) {
+                unlinkSync(`${stemTo}.d.ts.map`);
+            }
             renameSync(`${stemFrom}.cjs`, `${stemTo}.js`);
             renameSync(`${stemFrom}.d.cts`, `${stemTo}.d.ts`);
         }
